src/types/Error.ts

```diff
@@ -0,0 +1,6 @@
+export enum Error {
+  NONE = '',
```

Comment:
Consider renaming the enum to 'ErrorType' or 'ErrorMessages' for better clarity.